### ISSUE TITLE
I can not run LGNGUARD

### ISSUE THREAD

**Issue by changwenchi** [DiorsLi/RS_Zoos#1]
why i can not run LGNGUARD, Are there any environmentall requirements,the error occur in the follwing code:

def attention_coef(self, embedding_k, adj, memory):
        size = memory.shape
        emb_copy = embedding_k.cpu().data.numpy()#alpha uvk
        coef_matrix = torch.nn.functional.cosine_similarity(X=emb_copy, Y=emb_copy)
        row, col = adj.nonzero()

        coef = lil_matrix(size, dtype=np.float32)
        coef[row, col] = coef_matrix[row, col] 
        coef = normalize(coef, axis=1, norm='l1')
